<execution>LIFO</execution> </controls> <datasets> 
xmlns="uri:oozie:coordinator:0.2"> <controls> <timeout>10</timeout> <concurrency>2</concurrency> 
conf
</input-events> 
appPath
sc
</property></configuration> </workflow> </action> </coordinator-app>
writeToFile
appXml
<coordinator-app name="NAME" frequencyERROR="10" start="2009-02-01T01:00Z" end="2009-02-03T23:59Z" timezone="UTC" 
<property> <name>inputB</name> <value>${coord:dataOut('LOCAL_A')}</value> 
file://
getTestCaseDir
UNIT_TESTING
<dataset name="local_a" frequency="120" initial-instance="2009-02-01T01:00Z" 
<output-events> <data-out name="LOCAL_A" dataset="local_a"> 
timezone="UTC"> <uri-template>file:///tmp/coord/workflows/${YEAR}/${DAY}</uri-template> </dataset> 
coordinator.xml
set
<configuration> <property> <name>inputA</name> <value>${coord:dataIn('A')}</value> </property> 
<dataset name="a" frequency="60" initial-instance="2009-02-01T01:00Z" 
call
OozieClient
fail
testSchemaError
getTestUser
</datasets> <input-events> 
<instance>${coord:current(-1)}</instance> </data-out> </output-events> <action> <workflow> <app-path>hdfs:///tmp/workflows/</app-path> 
Exception expected if schema has errors!
<data-in name="A" dataset="a"> <instance>${coord:latest(0)}</instance> </data-in>  
File
